web/source/text/prediction/modelManager.ts

```diff
@@ -28,7 +28,7 @@ namespace com.keyman.text.prediction {
         }
       }
 
-      let lgCode = kbdInfo['languageCode'].toLowerCase();
+      let lgCode = kbdInfo['languageCode']?.toLowerCase();
```

Comment:
This seems like it might transfer errors further into the function? What is the behaviour if we end up with `this.languageModelMap[undefined]` on line 32? It seems to me that we should be avoiding trying to pass an `undefined` or `null` model to `loadModel`, given it does this:

https://github.com/keymanapp/keyman/blob/65770747a3c71ceaec99758c1e2a64f158a69a7a/common/web/input-processor/src/text/prediction/languageProcessor.ts#L107-L109

Reply:
`this.languageModelMap[undefined]` should return either `undefined` or `null`.  I could add a check in model registration to enforce this; we really should be filtering to prevent language-tagless models, I suppose.

Reply:
Yeah, looking at this code today, I think I must have been a bit sleepy when I wrote my previous comment. Looks fine today!